test/chi/custom-elements/dropdown.pug

```diff
@@ -4,22 +4,44 @@ title: Dropdown
 script(type='module' src='../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
 script(nomodule='' src='../../js/ce/ux-chi-ce/ux-chi-ce.js')
 
+.-mt--5(style='width: 500px; height: 500px;')
+  span.-text--h2=`Base`
+  .-w--100.-h--100.-d--flex.-align-items--center.-justify-content--center
+    chi-dropdown(data-cy="base-dropdown" active='false' button=`Base`)
+      a.chi-dropdown__menu-item(href='#' slot='menu') Menu item 1
+      a.chi-dropdown__menu-item(href='#' slot='menu') Menu item 2
+
+.-mt--5(style='width: 500px; height: 500px;')
+  span.-text--h2=`Hover`
+  .-w--100.-h--100.-d--flex.-align-items--center.-justify-content--center
+    chi-dropdown(data-cy="hover-dropdown" active='false' hover='true' button=`Hover`)
+      a.chi-dropdown__menu-item(href='#' slot='menu') Menu item 1
+      a.chi-dropdown__menu-item(href='#' slot='menu') Menu item 2
+
+.-mt--5(style='width: 500px; height: 500px;')
+  span.-text--h2=`Animate`
+  .-w--100.-h--100.-d--flex.-align-items--center.-justify-content--center
+    chi-dropdown(data-cy="animate-dropdown" active='false' button=`Animate` animate-chevron='true')
+      a.chi-dropdown__menu-item(href='#' slot='menu') Menu item 1
+      a.chi-dropdown__menu-item(href='#' slot='menu') Menu item 2
+
 - var positions = ['top', 'right', 'bottom', 'left', 'top-start', 'top-end', 'right-start', 'right-end', 'bottom-start', 'bottom-end', 'left-start', 'left-end'];
 - var methods = ['toggle', 'show', 'hide']
+- var events = ['show', 'hide']
```

Comment:
@soujanreddy in order to be consistent with the rest of tests we have in production, `vars` should be declared at the top of the page. 
Examples:
Date picker: https://github.com/CenturyLink/Chi/blob/master/test/chi/custom-elements/date-picker.pug
Drawer: https://github.com/CenturyLink/Chi/blob/master/test/chi/custom-elements/drawer.pug